packages/components/src/Notification/Notification.component.js

```diff
@@ -18,9 +20,11 @@ export function CloseButton({ notification, leaveFn }) {
 				theme['tc-notification-close'],
 				'.tc-notification-close',
 			)}
+			aria-label={t('NOTIFICATION_CLOSE', { defaultValue: 'Close notification' })}
```

Comment:
Are we sure `t` will always be defined, and that this will not end with `undefined is not a function` ? Especially since `t` is a new prop

Reply:
I connect it to the translation in the lines just after `export const CloseButton = translate(I18N_DOMAIN_COMPONENTS)(CloseButtonComponent);`

Reply:
Can we not use defaultProps and `getDefaultT` like many other functions? If we test the component without the HOC (with `WrappedComponent`) it would be an example of t as undefined.

Reply:
Ok, added the default props